src/utils/PortalAPI.ts

```diff
@@ -59,6 +59,19 @@ export class PortalAPI {
       console.warn('Criterion answer could not be processed.')
     }
   }
+
+  static async submitTermsAgree(participantID: string) {
+    const jsonData = JSON.stringify({ participant: participantID })
+    const response = await PortalAPI.executeAPIRequest(
+      'terms-and-conditions',
+      jsonData,
+    )
+
+    // If validation error
+    if (response.status === 400) {
+      console.warn("Could not agree to T&C's")
+    }
```

Comment:
We should think about what should happen in case the server errors out for different reasons (e.g. 504 gateway timeout.) Maybe an alert or toast?

Reply:
I agree, Might be worth adding a ticket to the backlog?